packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -1,5 +1,13 @@
 <template>
```

Comment:
If you are guests actions show you are not allowed to take, such as editing security settings, moving/adding projects. Members are hidden but the toggle to collapse is still shown. Maybe remove the toggle when there is no content

<img width="1115" alt="Screenshot 2024-12-18 at 17 07 57" src="https://github.com/user-attachments/assets/2518e197-3383-43ba-b2df-67cdbde00bc2" />